### ISSUE TITLE
Comment in protocol list changes indentation

### ISSUE THREAD

**Issue by vincentisambart** [nicklockwood/SwiftFormat#143]
```swift
class MyClass: ParentClass,
    Protocol1,
    // Some long remark about Protocol2
    Protocol2 {
    func foo() {
    }
}
```
gets formatted into:
```swift
class MyClass: ParentClass,
    Protocol1,
    // Some long remark about Protocol2
    Protocol2 {
        func foo() {
        }
    }
```

I would have expected the formatting not to change. The last `}` being indented feel pretty strange. What do you think?

**Comment by nicklockwood** [comment]
@vincentisambart yeah, that looks like a bug. I have special cases for wrapped if/guard statements - it looks like wrapped class definitions should be using the same approach.

**Comment by nicklockwood** [comment]
Fixed in 0.29.0

**Comment by vincentisambart** [comment]
Thank you!
